manim/scene/graph_scene.py

```diff
@@ -101,7 +101,7 @@ def setup_axes(self, animate=False):
             color=self.axes_color,
             include_tip=self.include_tip,
             add_start=self.x_add_start,
-            add_end=self.x_add_end,
+            add_end=self.x_add_end
```

Comment:
Question: did you do this manually or did `black` do this? It is my understanding that black prefers trailing commas in these cases. Please make sure to run `black` locally before submitting a PR.

Reply:
I did this manually.
The next commit will be formated